examples/tutorials/03_ready_to_use_model_torch.ipynb

```diff
@@ -18,10 +18,8 @@
     "import warnings\n",
```

Comment:
<p>Explain this chunck.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>